### PR TITLE
Fix doxygen typo and other various typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: rtengine/color.h [hunk 1]
```diff
@@ -1372,7 +1372,7 @@ class Color
     * @param HH hue before [-PI ; +PI]
     * @param Chprov1 chroma after [0 ; 180 (can be superior)]
     * @param CC chroma before [0 ; 180]
-    * @param corectionHuechroma hue correction depending on chromaticity (saturation), in radians [0 ; 0.45] (return value)
+    * @param correctionHuechroma hue correction depending on chromaticity (saturation), in radians [0 ; 0.45] (return value)
```
Comment (Floessie) [inline rtengine/color.h]: It's even `correctionHueChroma`.

**Review by Floessie — CHANGES_REQUESTED** [review]
Thanks, @luzpaz. There's one little change needed.

File: rtengine/color.h (hunk 1 above)
Comment (luzpaz) [reply rtengine/color.h]: Fixed: c5d2e00

**Comment by luzpaz** [comment]
Correct requested feedback + found 2 more typos 7f425e7

**Comment by Floessie** [comment]
@luzpaz Could you give current `dev` another sweep?

**Comment by luzpaz** [comment]
@Floessie Done, please see https://github.com/Beep6581/RawTherapee/pull/5804

**Pull request merged**